Recrutify/Recrutify.DataAccess/Models/ScheduleSlot.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class ScheduleSlot
+    {
+        public DateTime AvailableTime { get; set; }
+
+        public ParticipantSlot ParticipantSlot { get; set; }
```

Comment:
AssignedCandidate, and don't forget to rename all fields in scripts too!

Reply:
done